### PR TITLE
Chapter 6: Modeling User

### PR DESCRIPTION
**Pull request by tranghoangthu812102**
Trả lời câu hỏi: https://docs.google.com/document/d/1EPbBUAInDHD7fV2NDPeGVWlkLrS0T5LfXGgq_jTFPys/edit
Screenshot from 2021-08-02 06-10-40

### CODE REVIEW

**Comment by tranghoangthu812102** [comment]
ready

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  validates :email, presence: true,
+                    length: {
+                      maximum: Settings.validation.email.max_length,
+                      minimum: Settings.validation.email.min_length
+                    },
+                    format: {with: Settings.validation.email.format_regex},
+                    uniqueness: {case_sensitive: false}
+
+  has_secure_password
+  validates :password, presence: true,
```
Comment (ngocvt-0484) [inline app/models/user.rb:15]: cho validate lên dòng 14

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  validates :email, presence: true,
+                    length: {
+                      maximum: Settings.validation.email.max_length,
+                      minimum: Settings.validation.email.min_length
+                    },
+                    format: {with: Settings.validation.email.format_regex},
+                    uniqueness: {case_sensitive: false}
+
+  has_secure_password
```
Comment (ngocvt-0484) [inline app/models/user.rb]: cho xuống dưới

**Comment by tranghoangthu812102** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**